paasta_tools/kubernetes/bin/paasta_secrets_sync.py

```diff
@@ -81,6 +82,13 @@ def parse_args() -> argparse.Namespace:
         dest="namespace",
         help="Overwrite destination namespace for secrets",
     )
+    parser.add_argument(
+        "-vtf",
```

Comment:
(personal opinion): multi-char short options always look a little weird to me (they usually make me think of linux cli tools where you can chain multiple short options using 1 `-`) - thoughts on having the short option here be `-t` instead?